src/Components/Blazor/Build/src/targets/All.targets

```diff
@@ -6,8 +6,8 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <BlazorToolsDir Condition="'$(BlazorToolsDir)' == ''">$(MSBuildThisFileDirectory)../tools/</BlazorToolsDir>
-    <BlazorBuildExe>dotnet &quot;$(BlazorToolsDir)Microsoft.AspNetCore.Blazor.Build.dll&quot;</BlazorBuildExe>
+    <BlazorToolsDir Condition="'$(BlazorToolsDir)' == ''">$(MSBuildThisFileDirectory)..\tools\</BlazorToolsDir>
```

Comment:
There is some possibility we needed the `/` due to passing the path into some Mono tooling that only understood those characters. Hopefully that's not the case and the MSBuild standardization is OK.

But if the build starts failing with this change, I hope this is a useful clue why!

Reply:
So far things look fine. 🤷‍♀